docs/DESIGN_TOKENS.md

```
@@ -0,0 +1,288 @@
+
+| Token | Light Mode | Dark Mode |
+|-------|-----------|-----------|
+| `--sidebar-background` | 246 247 248 | 9 9 9 |
+| `--sidebar-foreground` | 12 12 12 | 244 244 245 |
+| `--sidebar-primary` | 12 12 12 | 244 244 245 |
+| `--sidebar-accent` | 245 246 248 | 32 33 37 |
+| `--sidebar-border` | 229 231 235 | 255 255 255 / 0.08 |
+
+**Usage in components:**
+```tsx
+<aside className="bg-sidebar text-sidebar-foreground">
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)